config/internal/configsource/manager.go

```diff
@@ -182,14 +191,22 @@ func NewManager(_ *configparser.ConfigMap) (*Manager, error) {
 	}, nil
 }
 
-// Resolve inspects the given config.Parser and resolves all config sources referenced
-// in the configuration, returning a config.Parser fully resolved. This must be called only
-// once per lifetime of a Manager object.
+// Resolve inspects the given configparser.Parser and resolves all config sources referenced
```

Comment:
I think `configparser.Parser` was renamed to `configparser.ConfigMap` so the comment probably needs to be updated.